lms/djangoapps/teams/tests/test_views.py

```diff
@@ -821,6 +821,16 @@ def test_member_trying_to_get_private_topic(self):
         self.assertEqual('private_topic_1_id', result['results'][0]['topic_id'])
         self.assertNotEqual([], result['results'])
 
+    def test_course_staff_getting_information_on_private_topic(self):
+        """
+        Verifies that when an admin browses to a private team set,
+         information about the teams in the teamset is returned even if the admin is not in any teams.
+        """
+        result = self.get_teams_list(data={'topic_id': 'private_topic_1_id'},
+                                     user='course_staff')
+        self.assertEqual(2, len(result['results']))
+        self.assertNotEqual([], result['results'])
```

Comment:
this last line seems unnecessary. maybe assert the team names? or just remove this line

Reply:
If you wanna be fancy you could combine test_course_staff_getting_information_on_private_topic test_member_trying_to_get_private_topic and test_non_member_trying_to_get_private_topic into one ddt test?